server/app.js

```diff
@@ -0,0 +1,17 @@
+import '@babel/polyfill';
+import express from 'express';
+import cors from'cors';
+import db from './database/models/index';
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+const port = process.env.PORT || 2000;
+
+app.listen(port, () => {
+  // eslint-disable-next-line no-console
+  console.log(`App is listen on ${port}`);
+  db.sequelize.sync();
```

Comment:
Why are you doing this? - `db.sequelize.sync()` and also you left out using `express.urlencoded`

Reply:
From the material you sent to me yesterday, I found out that body-parser is not needed because I was not making any request at the moment.

The db.sequelize.sync() function is used to show that migration has been established in the database.

Reply:
I don't think you need to use `db.sequelize.sync` if you want to run migrations when starting up  the server you can create an npm script to do that and add it the `npm start` command for starting the server @amaechi-chuks

Reply:
Noted.